doc/sphinx-guides/source/_static/api/add-license.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "CC-BY-4.0",
```

Comment:
I think the URL is really the primary identifier and the 'name' is intended for display rather than as a second less specific way to identify the license globally. So - I think I agree that the non-hyphenated versions make more sense and we can change those (unless someone else has a concern about that?). 

(Note - since these are configured via API, installations that prefer some other user-visible label (or want to have them in another language, etc.), they can still do that. The interoperability is intended to be by URL - if that matches, it's the same license.

Reply:
I have no objection to using a URL as a primary identifier but are our URLs aligned with SPDX, for example? It seems like not (CC-BY-4.0 example):

- this PR: http://creativecommons.org/licenses/by/4.0
- SPDX: https://spdx.org/licenses/CC-BY-4.0.html

In short, if we're going to use URLs as identifiers it would be nice to see a canonical list of these URLs somewhere.

Reply:
The URLs used now are what Creative Commons has assigned to their licenses. Their page metadata include attributes like `about="http://creativecommons.org/licenses/by/4.0" `for example. While SPDX has assigned alternate URLs, I don't think it would be best practice to not follow Creative Commons. Since licenses are configurable, installations could choose to do that if they wanted to though.